doc/whats_new/v0.22.rst

```diff
@@ -57,16 +57,22 @@ Changelog
   ``decision_function_shape='ovr'``, and the number of target classes > 2.
   :pr:`12557` by `Adrin Jalali`_.
 
-
 :mod:`sklearn.cluster`
 ..................
 
 - |Enhancement| :class:`cluster.SpectralClustering` now accepts a ``n_components`` 
   parameter. This parameter extends `SpectralClustering` class functionality to
   match `spectral_clustering`.
   :pr:`13726` by :user:`Shuzhe Xiao <fdas3213>`.
+
+:mod:`sklearn.feature_selection`
+................................
+- |Fix| :class:`feature_selection.VarianceThreshold` now uses range
+  rather than variance when :term:`threshold` is 0. This prevents constant
```

Comment:
I should have picked these things up before. `:term:` can only be used when there is an entry in the glossary.

it may also be more helpful to phrase this with respect to the user: "Fixed a bug where VarianceThreshold with threshold=0 did not remove constant features due to numerical instability."